### PR TITLE
Using Colab to upload files to GCS

### PR DESCRIPTION
**Pull request by GbotemiB**
The NYC dataset can be uploaded faster to gcs using colab.

@alexeygrigorev please check it out.

### CODE REVIEW

File: week_3_data_warehouse/extras/README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@ Quick hack to load files directly to GCS, without Airflow. Downloads csv files f
 
 1. Install pre-reqs (more info in `web_to_gcs.py` script)
 2. Run: `python web_to_gcs.py`
+
+Another method is to use the web_to_gcs_on_colab.ipynb file on colab to upload using colab
```
Comment (Victoriapm) [inline week_3_data_warehouse/extras/README.md:6]: is it another script or the same just running on colab?

Comment (GbotemiB) [reply week_3_data_warehouse/extras/README.md:6]: Yes it is another script @Victoriapm

**Comment by alexeygrigorev** [comment]
@AnkushKhanna can you please take a look?

**Comment by alexeygrigorev** [comment]
Closing it as quite old

**Pull request closed without merging**